python/python/bystro/ancestry/define_callset.py

```diff
@@ -0,0 +1,141 @@
+"""Calculate intersection of Illumina and Affymetrix chips."""
+
+import logging
+import re
+
+import pandas as pd
+import tqdm
+from liftover import get_lifter
+
+from bystro.ancestry.asserts import assert_equals, assert_true
+from bystro.ancestry.train import DATA_DIR, INTERMEDIATE_DATA_DIR
+from bystro.ancestry.train_utils import is_autosomal_variant
+
+logger = logging.getLogger(__name__)
+converter = get_lifter("hg19", "hg38")
+
+ILLUMINA_FILEPATH = DATA_DIR / "Human660W-Quad_v1_H.csv"
+AFFYMETRIX_FILEPATH = DATA_DIR / "Axiom_PMRA.na35.annot.csv"
+
+
+def get_watson_crick_complement(base: str) -> str:
+    """Calculate Watson-Crick Complement."""
+    wc_dict = {"A": "T", "T": "A", "G": "C", "C": "G"}
+    return wc_dict[base]
+
+
+def liftover_38_from_37(variant: str) -> str | None:
+    """Liftover a variant to genome build 38 from 37."""
+    chrom, pos, ref, alt = variant.split(":")
+    if "MT" in chrom:
```

Comment:
It looks like there already shouldn't be MT in either illumina.variants or affymetrix.variants workflow when it gets to this part so I think this step might be redundant? Or it might make sense to include the is_autosomal_variant step here.

Reply:
yes, took the redundant check out as per https://github.com/bystrogenomics/bystro/pull/154#discussion_r1265805259, thanks!